### PR TITLE
Issue/6438 login prologue analytics

### PR DESCRIPTION
**Pull request by theck13**
### Fix
Add analytics to login prologue to track when users page through the screens as described in https://github.com/wordpress-mobile/WordPress-Android/issues/6438.

### Test
1. Set breakpoints on each case of [login prologue paging](https://github.com/wordpress-mobile/WordPress-Android/blob/cf4f570749c1d11a71c47bcda86715253e7b4dc9/WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginPrologueFragment.java#L53-L67).
2. Swipe through login prologue screens.
3. Notice breakpoints are hit.
4. Notice [analytics events](https://github.com/wordpress-mobile/WordPress-Android/blob/cf4f570749c1d11a71c47bcda86715253e7b4dc9/libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java#L691-L700) are tracked.

### CODE REVIEW

**Comment by hypest** [comment]
Hey @theck13 , I think https://github.com/wordpress-mobile/WordPress-Android/issues/6438 is a duplicate of https://github.com/wordpress-mobile/WordPress-Android/issues/6542 and the latter is already addressed so, is there a need for the extra handling?

**Comment by theck13** [comment]
I didn't even notice the duplicate.  Thanks for pointing that out, @hypest!  Eric and I [discussed](https://a8c.slack.com/archives/C02QANACA/p1502988564000432) the intention of https://github.com/wordpress-mobile/WordPress-Android/issues/6542 and decided those changes should only track when the screens are swiped and not which screens are shown.  These changes track which screens are shown.  Maybe I misunderstood the issue description.  Do you think that's something we should track, @aerych?

**Comment by aerych** [comment]
>  I think #6438 is a duplicate of #6542 

Doh!  My bad :( 

> These changes track which screens are shown. [...]  Do you think that's something we should track

Yes please. It might be interesting to know how often they are *all* viewed.

File: WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginPrologueFragment.java [hunk 1]
```diff
@@ -48,6 +49,24 @@ public void onPageScrolled(int position, float positionOffset, int positionOffse
 
             @Override
             public void onPageSelected(int position) {
+                switch (mPagerAdapter.getTag(position)) {
```
Comment (hypest) [inline WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginPrologueFragment.java]: Hey @theck13 , what do you think about making `getTag()` a static method (and the `mTags` field static as well)? The `mPagerAdapter` reference won't be needed in that case.

Comment (theck13) [reply WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginPrologueFragment.java]: That works for me.  I updated the `getTag()` method and `mTags` array in bc44e1941a3c485d407dc28230ee22f301e4a052.

**Comment by hypest** [comment]
LGTM!

**Pull request merged**